static/js/router.js

```diff
@@ -26,7 +26,7 @@
         templateUrl: 'templates/partials/messages.html'
       })
       .state('messages.detail', {
-        url: '/:id',
+        url: '/:type/:id',
```

Comment:
This will break existing URLs. We could try and manage backwards compatibility but it's probably not worth it. What do you think?